babel.config.json

```diff
@@ -11,8 +11,10 @@
         "@babel/proposal-optional-chaining"
       ]
     }],
-    "@babel/preset-react",
+    ["@babel/react", { "runtime": "automatic" }],
     "@babel/typescript"
   ],
-  "plugins": []
+  "plugins": [
+    ["@babel/transform-runtime", { "useESModules": true }]
```

Comment:
This (+ `babelHelpers: 'runtime',` in the rollup config) will keep babel helpers outside our bundle, to avoid duplicate code in users' bundles.